doc/getting_started/installation_linux.rst

```diff
@@ -143,6 +143,12 @@ of obtaining a more recent one:
 You might also want to uninstall the CMake provided by your package manager to
 avoid conflicts.
 
+**A recent DTC version (1.5 or higher) is required**. Check what version you
```

Comment:
I have older DTC (Fedora 30) and I can compile zephyr just fine
```
$ dtc --version
Version: DTC 1.4.7
```

Reply:
You are right, fixed now!

Reply:
@galak is DTC 1.4.7 officially supported?

Reply:
The min version is 1.4.6, so anything newer works.